### PR TITLE
Add obsidian

### PR DESCRIPTION
**Pull request by wizzdom**
This PR adds a few things that make writing and maintaining docs much easier:
- initializes an obsidian vault in the `docs/` folder
- greatly expands upon the internal linking in the repo already, therefore making the docs site easier to navigate
- adds dedicated pages for each host like how it was done on the old docs (unfinished at the moment)

### CODE REVIEW

File: docs/.obsidian/workspace.json
Comment (DistroByte) [inline docs/.obsidian/workspace.json:1]: I don't think we need to commit this file

**Review by DistroByte — CHANGES_REQUESTED** [review]
Aside from that workspace file, all LGTM

File: docs/.obsidian/workspace.json
Comment (wizzdom) [reply docs/.obsidian/workspace.json:1]: That makes sense.
Removed and gitignored :+1:

**Review by DistroByte — APPROVED** [review]
LGTM

**Pull request merged**